### ISSUE TITLE
Small bug with refreshing - expansion icon not always there

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#889]
This is with 3.6.0.

I needed to refresh some historical partitions and did them in small batches. I've seen this happen before - the expansion icon doesn't show up so I cannot see the progress. The first one (completed at the top) never gave indication as to where it was. The second group did and I can expand it and see the progress of the partitions. 

![image](https://user-images.githubusercontent.com/45298358/234718454-c6d5caef-ed3c-42aa-88a4-966a2d8b90b9.png)

**Comment by otykier** [comment]
Yeah, we've also seen this. Sometimes, the AS trace initiated by Tabular Editor to collect the progress messages, fails to bind against the Power BI XMLA endpoint. When that happens, we don't get the progress notifications, and that explains why we're not able to show anything. We'll review the code to see if there's anything we can do differently to improve the stability of this. Thanks!
